timemachine/fe/rbfe.py

```diff
@@ -178,8 +178,8 @@ def setup_initial_states(st, host_config, temperature, lambda_schedule, seed):
 
     initial_states = []
 
-    # check that the lambda schedule is monotonically increasing.
-    assert np.all(np.diff(lambda_schedule) > 0)
+    # check that the lambda schedule is monotonically increasing or decreasing.
+    assert np.all(np.diff(lambda_schedule) > 0) or np.all(np.diff(lambda_schedule) < 0)
```

Comment:
Possible bug: This looks like it weakens an assertion when used for RBFE (unless RBFE estimate is already invariant to reversal of lambda schedule).

(Consider removal, or specializing to assert one order for RBFE and the opposite order for AHFE, or re-introducing the original assertion in a still-RBFE-specific codepath.)

Reply:
Definitely not intended, reverting.